packages/atomic/src/pages/examples/commerce-website/engine.mjs

```diff
@@ -25,55 +20,36 @@ export const commerceEngineConfig = {
       referrer: document.referrer,
     },
   },
-};
-
-export const setupRecommendationsMostPurchased = (engine) => {
-  const recommendationsMostPurchased = buildRecommendations(engine, {
-    options: {
-      slotId: '573ae88e-37ae-456f-a949-829164ad7a84',
-    },
-  });
-  recommendationsMostPurchased.refresh();
-  return recommendationsMostPurchased;
-};
-
-export const setupRecommendationsMostViewed = (engine) => {
-  const recommendationsMostViewed = buildRecommendations(engine, {
-    options: {
-      slotId: 'e098ca76-e7de-4488-b818-c87fbce35542',
-    },
-  });
-  recommendationsMostViewed.refresh();
-  return recommendationsMostViewed;
-};
-
-export const setupProductListing = (engine) => {
-  const productListing = buildProductListing(engine);
-  productListing.refresh();
-  return productListing;
-};
-
-export const setupCart = (engine) => {
-  const cart = buildCart(engine);
-  return cart;
-};
-
-export const renderProducts = (elem, title, products) => {
-  if (!products.length) {
-    document.querySelector(elem).innerHTML = 'No products found.';
-    return;
-  }
-  document.querySelector(elem).innerHTML =
-    `<h2>${title}</h2>` +
-    products
-      .map((product) => {
-        return `<span><a href='${product.clickUri}'><img width="100px" src='${product.ec_images[0]}'>${product.ec_name}</a></span>`;
-      })
-      .join('');
-
-  return products
-    .map((product) => {
-      return `<div>${product.name}</div>`;
-    })
-    .join('');
+  cart: {
```

Comment:
After chatting with @samisayegh, @sallainCoveo and @BenKandelaars, we'll want to change the cart's state structure to better support Coveo's needs.

Not blocking but just a heads up 😄 You should get a meeting invite about this soon 😅